keyboards/kingly_keys/romac_plus/romac_plus.c

```diff
@@ -0,0 +1,4 @@
+#include "romac_plus.h"
+
+#ifdef __attribute__((weak)) OLED_DRIVER_ENABLE
```

Comment:
How should this be formatted?

Reply:
This shouldn't be needed at all, actually.